guide/glossary.md

```diff
@@ -251,19 +251,37 @@ A standard for multi-signature that uses Schnorr signatures. Previously, the mor
 
 ### Node
 
-Node refers to [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) that participates in the bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#hardware-wallets' | relative_url }}).
+A node refers to a Bitcoin client that participates in the Bitcoin network.
 
-#### Full node
+#### Full nodes
 
-A node that fully verifies all transactions and rules of the Bitcoin network.
+Full nodes are Bitcoin clients that fully verify and enforce the rules of the Bitcoin network. An example of a rule is that there will never be more than 21 million bitcoin.
```

Comment:
I think we need to explain somewhere (not necessarily here) what it means that the node verifies and enforces rules. Breaking the topic of nodes out into a separate page will allow us to do that.

Reply:
Also think that the visual hierarchy is problematic as H3 and H4 headers are really hard to distinguish. Maybe go for bullets or H5 instead?

Reply:
A nodes chapter maybe ;)

Reply:
Yeah, we should wrap up this PR and start working on #78.